libyara/parser.c

```diff
@@ -548,14 +549,35 @@ int yr_parser_reduce_string_declaration(
       goto _exit;
   }
 
+  // base64 and nocase together is not implemented.
+
+  if (MODIFIER_IS_ANY_BASE64(modifier) && modifier.flags & STRING_GFLAGS_NO_CASE)
```

Comment:
I see the point of using the `MODIFIER_IS_ANY_BASE64` macro, as the are multiple places where we check for both flags, but at the same time, given that the rest of the condition is about checking flags too, I think I prefer uniformity over brevity, like in:

```c
if (modifier.flags & STRING_GFLAGS_BASE64 && 
    modifier.flags & STRING_GFLAGS_BASE64_WIDE && 
    modifier.flags & STRING_GFLAGS_NO_CASE)
{
...
}
```

Reply:
Done! I've removed all references to `MODIFIER_IS_ANY_BASE64` everywhere.